.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -18,6 +18,31 @@ body:
         - label: I agree to follow this project's [Code of Conduct](https://github.com/golangci/golangci-lint?tab=coc-ov-file)
           required: true
 
+  - type: dropdown
+    id: install
+    attributes:
+      label: How did you install golangci-lint?
+      options:
+        - I don't know
+        - Official GitHub Action
+        - Official binary
+        - Brew
+        - MacPorts
+        - Chocolatey
+        - Scoop
+        - Docker
+        - go install
+        - Tools pattern
+        - go tool
+        - AUR
+        - Nix
+        - Deb
+        - RPM
```

Comment:
I would mention local compilation involving custom-gcl way

Reply:
The custom binary is not strictly an installation because it requires installing golangci-lint before.